api/README.md

```diff
@@ -37,6 +37,12 @@ source .env
 python3 ./create_db.py
 ```
 
-This may change if we need to do more complex migrations. It's probably easier to just to drop the tables.
+You can do this with `psql $DATABASE_URL`, but don't do this on lab machines, as it'll leak the password :)
 
-You can do this with `psql $DATABASE_URL`, but don't do this on lab machines, as it'll leak the password :)
\ No newline at end of file
+## Prod DB """Migrations"""
+
+1. Setup `$DATABASE_URL`
```

Comment:
Probably worth stating that we need to get this from the Heroku Postgres Add-on web-ui

Reply:
This is documented above lol